pkg/util/kubernetes/apiserver/controllers.go

```diff
@@ -42,6 +43,9 @@ type ControllerContext struct {
 // StartControllers runs the enabled Kubernetes controllers for the Datadog Cluster Agent. This is
 // only called once, when we have confirmed we could correctly connect to the API server.
 func StartControllers(ctx ControllerContext) error {
+	if !cache.WaitForCacheSync(ctx.StopCh, ctx.InformerFactory.Core().V1().Nodes().Informer().HasSynced) {
```

Comment:
Does this work without `InformerFactory.Start()` not being called yet?

Reply:
it's a blocking call actually, even if put after the controllers start - I'm removing this for now, I'm adding a card to investigate